src/hpnmg/ProbabilityCalculator.h

```diff
@@ -33,7 +35,10 @@ namespace hpnmg {
 		pair<string, map<string, float>> distribution;
 	} singleDim;
 
-
+	struct  functionToDirectIntegrateMonteCarloParams{
```

Comment:
We should start adding more documentation:
- Data-type for the GSL monte carlo integrand's general purpose parameter.
- `unionOfPolytope` represents the exact domain of integration as union of all polytopes in the vector.
- `distributions` provides one pdf (or rather a parametrization thereof) for every dimension of the polytopes' vector space.